heuristic/classes/Solution.py

```diff
@@ -165,3 +165,38 @@ def to_file(self, location: str):
                     print(f"V{idx_route},{leg},S{idx_stack},{stack}", file=file)
 
         file.close()
+
+    def to_graph(self):
+        problem = Problem()
+
+        import networkx as nx
+        import matplotlib.pyplot as plt
+        import numpy as np
+        from sklearn.decomposition import PCA
+        from sklearn import manifold
+
+        pca = PCA(n_components=2)
+        coords = pca.fit_transform(problem.distances)
```

Comment:
This _might_ be TC, but I'm not sure about the output yet. If it looks good, it's fine by me - else simpler methods might be preferred.